service/src/test/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperServiceTest.java

```diff
@@ -752,6 +752,36 @@ void shouldSortDirectionsIntoSeparateEntriesInMapWhenManyAssignees() {
         }
     }
 
+    @Nested
+    class AddEmptyDirectionsForAssigneeNotInMap {
+
+        @Test
+        void shouldAddEmptyListValueWhenKeyNotPresentInMap() {
+            Map<DirectionAssignee, List<Element<Direction>>> map = new HashMap<>();
+
+            service.addEmptyDirectionsForAssigneeNotInMap(map);
+
+            assertThat(map).containsOnlyKeys(DirectionAssignee.values());
```

Comment:
You could loop over all the values and assert that each entry is an empty list, would make the test follow the name a bit closer

Reply:
I can add to the assert, sure